packages/vue-components/components/auth/lost-password/lost-password.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <generic-form
+      :fields
+      :loading
+      :buttons="{
+        submit: true,
```

Comment:
Isn't the _submit_ button `true` by default?

Reply:
That's right, but not the `cancel` one, so I need to provide the full object.